src/talk/renderer/UserStatus/userStatus.utils.ts

```diff
@@ -4,8 +4,8 @@
  */
 
 import type { ClearAtPredefinedConfig, PredefinedUserStatus, UserStatus, UserStatusStatusType } from './userStatus.types.ts'
-import moment from '@nextcloud/moment'
-import { translate as t } from '@nextcloud/l10n'
+import { t, getFirstDay } from '@nextcloud/l10n'
```

Comment:
Screenshot from `appData.userMetadata`, this results in empty `<html data-locale >` (when was not set on personal settings). Maybe it should defaults to selected language; here to 'de'?

![image](https://github.com/user-attachments/assets/63d09208-e94d-4116-9c48-0d3f1df2d785)

also custom `window.firstDay` from settings is not provided as in web, and always leads to default locale value:
- 'en' - Sunday
- 'de' - Monday

Reply:
> also custom `window.firstDay` from settings is not provided as in web

Yes, this info is not available for clients, so, using locale data

Reply:
> Screenshot from `appData.userMetadata`, this results in empty `<html data-locale >` (when was not set on personal settings). Maybe it should defaults to selected language; here to 'de'?

I cannot reproduce this state... Even for new accounts it's `en` initially. How do you unset it?

Anyway, Nextcloud defaults it to `'en'` in `@nextcloud/l10n`, so should be fixed there. But I'd fallback for the local device settings, not the language.

Reply:
`occ user:setting <userid> --delete core locale`. Then in web it defaults to `core.lang`

Reply:
Proposal: https://github.com/nextcloud-libraries/nextcloud-l10n/pull/864